clr-k8s-examples/vagrant.md

```diff
@@ -27,3 +27,21 @@ Note, vagrant installation steps were derived from:
 * https://computingforgeeks.com/using-vagrant-with-libvirt-on-linux/
 * https://computingforgeeks.com/install-latest-vagrant-on-ubuntu-18-04-debian-9-kali-linux/
 * https://github.com/vagrant-libvirt/vagrant-libvirt/blob/master/README.md
+
+## Install vagrant on Clear Linux
+
+On Clear Linux, run these commands
+```bash
+sudo wget https://github.com/AntonioMeireles/ClearLinux-packer/blob/master/extras/clearlinux/setup/libvirtd.sh
```

Comment:
no raw URL for this?

Reply:
can you please clarify more on raw URL.

Reply:
the second URL seems to point to raw version `https://raw.githubusercontent.com/` instead of rendered. Does this one not need to download from raw version?

Reply:
yes, will update the raw URL for the first one : https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/libvirtd.sh